internal/tenants/manager.go

```diff
@@ -15,10 +16,15 @@ type Manager struct {
 	timeout       time.Duration
 	tenantsMutex  sync.Mutex
 	tenants       map[int64]*tenantInfo
+	logger        zerolog.Logger
 }
 
 var _ pusher.TenantProvider = &Manager{}
 
+const (
+	secretsTimeout = 8 * time.Minute
```

Comment:
Why 8 minutes?

Reply:
> Why 8 minutes? I think it should be 7 if its 10 minus [maxScriptedTimeout](https://github.com/grafana/synthetic-monitoring-agent/blob/main/pkg/pb/synthetic_monitoring/checks_extra.go?rgh-link-date=2025-04-04T13%3A01%3A14Z#L151).

ah, good point. I remember you mentioned this, but I forgot why you said it should be less.